session/src/main/java/uk/gov/di/ipv/cri/common/api/service/SessionRequestService.java

```diff
@@ -90,6 +92,15 @@ private SessionRequest parseSessionRequest(String requestBody)
             sessionRequest.setState(jwtClaims.getStringClaim("state"));
             sessionRequest.setSubject(jwtClaims.getSubject());
 
+            if (jwtClaims.getClaims().containsKey(PERSISTENT_SESSION_ID)) {
```

Comment:
Update `SessionRequestServiceTest.java` to cover the new code

Reply:
Added field to the test shouldValidateJWTSignedWithECKey (via makeSessionRequestFieldValueAssertions). SignedJWTBuilder was modified to allow this.